Formula/func-e.rb

```diff
@@ -0,0 +1,21 @@
+class FuncE < Formula
+  desc "Easily run Envoy®"
```

Comment:
No need for emoji here.
```suggestion
  desc "Easily run Envoy"
```